erpnext/healthcare/doctype/appointment_type_service_item/appointment_type_service_item.json

```diff
@@ -48,7 +48,7 @@
    "fieldname": "inpatient_visit_charge",
    "fieldtype": "Currency",
    "in_list_view": 1,
-   "label": "Inpatient Visit Charge Item"
+   "label": "Inpatient Visit Charge"
```

Comment:
Change the modified timestamp in this JSON file so that the changes would be synced on migrating.